travis build failing

**Actual Behaviour**

travis build is failing 

**Expected Behaviour**

travis build should not fail

**Steps to reproduce it**

NA

**LogCat for the issue**

```Installing Solver for ConstraintLayout 1.0.2 in /usr/local/android-sdk/extras/m2repository/com/android/support/constraint/constraint-layout-solver/1.0.2
"Install Solver for ConstraintLayout 1.0.2" failed.
Checking the license for package ConstraintLayout for Android 1.0.2 in /usr/local/android-sdk/licenses
License for package ConstraintLayout for Android 1.0.2 accepted.
Preparing "Install ConstraintLayout for Android 1.0.2".
"Install ConstraintLayout for Android 1.0.2" ready.
Finishing "Install ConstraintLayout for Android 1.0.2"
Installing ConstraintLayout for Android 1.0.2 in /usr/local/android-sdk/extras/m2repository/com/android/support/constraint/constraint-layout/1.0.2
"Install ConstraintLayout for Android 1.0.2" failed.
FAILURE: Build failed with an exception.
* What went wrong:
A problem occurred configuring project ':app'.
> Failed to install the following SDK components:
  [ConstraintLayout for Android 1.0.2, Solver for ConstraintLayout 1.0.2]
  Please install the missing components using the SDK manager in Android Studio.
```

**Screenshots of the issue**

NA

**Would you like to work on the issue?**

Yes

--- a/build.gradle
+++ b/build.gradle
@@ -8,7 +8,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.android.tools.build:gradle:2.3.3'
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
     }

